Makefile

```diff
@@ -10,6 +10,10 @@ ifdef TEST
 	DOCEST_TEST=validator/rules_src/$(TEST).py
 endif
 
+ifdef PEP8
```

Comment:
This features about style is good! Just let them know to pip install pep8 and black in advance.
Code change needed.

Reply:
I agree, we should inform users how to install packages, but an issue is already open for CONTRIBUTE.md (#7) which will include all of make commands (all of installations are done using makefile) . So this feature needs separate PR.

Reply:
Noted, thanks. Agreed this one is separate one.